### PR TITLE
Refactor to func comp with hook

### PR DESCRIPTION
**Pull request by Apatakiti**
I'm done with this part
### Project requirements

- Refactor the Calculator component (components/Calculator.js) from a class-based component to a functional component with hooks.
- After the refactor is done, test the app in the browser. It should keep the same functionality.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,8 +97,15 @@ you can have a copy of this project by cloning this repo to your local machine
...
 
+This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+
+### `npm run build` fails to minify
+
+This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+
 👤 **Author**
 
 - GitHub: [@githubhandle](https://github.com/Ademola-coding)
```
Comment (muneebulrehman) [inline README.md:78]: Kindly update your README file as per [this](https://github.com/microverseinc/readme-template/blob/master/README.md) template.  You can customize your README file however you want, but the main headers from this provided template README file should be present. Also, kindly remove the sections which are not relevant to your project like Getting Started with Create React App, Available scripts, etc.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @Ademola-coding , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Descriptive PR

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Apatakiti) [reply README.md:78]: THANKS

**Review by Deepakdanger — APPROVED** [review]
Hi @Ademola-coding, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**